app/routes/groups_.new.tsx

```diff
@@ -0,0 +1,67 @@
+import { Card } from '~/components/ui/containers';
+
+export default function GroupNew() {
+  return (
+    <div className="grow py-20 lg:py-40 bg-secondary w-full flex flex-col items-center justify-center">
+      <div className="flex w-full items-center justify-center flex-col">
+        <div className="max-w-[900px] w-full flex flex-col gap-3">
+          <h1 className="text-black font-bold text-4xl">Create New Group</h1>
```

Comment:
We could create re-usable components for `h1` and `h2` here since they are reused across different pages in our designs.

Reply:
I'll add a task for this